src/exporter/util.h

```diff
@@ -20,3 +20,6 @@ class BlockTimer {
 bool string_is_digit(std::string s);
 std::string read_file_to_string(std::string path);
 std::string get_hostname(std::string path);
+
+// Must be kept in sync with promethize() in src/pybind/mgr/prometheus/module.py
```

Comment:
Move this comment to the definition -- it's much more likely to be noticed there.  If someone tweaks the body without needing to tweak the signature chances are they won't even open the header file.

Reply:
And please add a corresponding comment to `src/pybind/mgr/prometheus/module.py` here, in the same commit.